source/common/config/delta_subscription_impl.cc

```diff
@@ -32,6 +32,10 @@ void DeltaSubscriptionImpl::start(const std::set<std::string>& resources) {
   stats_.update_attempt_.inc();
 }
 
+void DeltaSubscriptionImpl::addResourceAliases(const std::set<std::string>& aliases) {
```

Comment:
Are these aliases going to be persistently monitored by the Envoy xDS client, or is it a one time thing? "add" sounds like the former, "request resolution" sounds like the latter.

From reading the code a bit, it looks like the latter. I would suggest naming this something like requestAliasResolution. (lol I just scrolled down a tiny bit and noticed that that's what you call it in the next level; perfect! Great minds think alike. Yeah, standardize on 'request' I would say.)

Reply:
done.